_episodes/05-function-style.md

```diff
@@ -14,41 +14,58 @@ keypoints:
 ---
 
 # Adding a function to our package
-We will now add a new function to our Python package. Open your `molssi_math.py` module in a text editor. Currently, it should only have the default `canvas` function written by the CMS CookieCutter.
+We will now add a new function to our Python package. Create a new folder named `io` under `molecool` directory and add a file named `pdb.py`. Open your `molecool/io/pdb.py` module in a text editor. Currently, it should only have the default `canvas` function written by the CMS CookieCutter.
```

Comment:
From the previous episode they will already have the structure decided (so no need to create the new folder and give the file a name). They will just need to open the file `pdb.py`. This module will not have the `canvas` function.